src/components/Landing/QAs.tsx

```diff
@@ -19,32 +34,74 @@ export default function QAs() {
     {
       q: t`What does Juicebox cost?`,
       a: [
-        t`Juicebox is an open protocol on Ethereum that is funded using Juicebox itself. You can check out the contractualized budget specs at https://juicebox.money/#/p/juicebox.`,
-        t`5% of all money made by projects is sent to help pay for the maintenance and development of Juicebox itself. In exchange, projects get Juicebox tokens ($JBX), which will be worth more as the ecosystem grows over time.`,
+        <Trans>
+          Juicebox is an open protocol on Ethereum that is funded using Juicebox
+          itself. You can check out the contractualized budget specs{' '}
+          <a
+            href="https://juicebox.money/#/p/juicebox"
+            rel="noreferrer"
+            target="_blank"
+          >
+            {' '}
+            here
+          </a>
+          .
+        </Trans>,
+        <Trans>
+          Projects building on Juicebox pay a 5% JBX membership fee from
+          withdrawn funds into the JuiceboxDAO treasury. Projects can then use
+          their JBX to participate in the governance of JuiceboxDAO, as well as
+          redeem from its growing {overflowVideoLink('overflow')}. The fee is
+          also subject to change via JBX member votes.
+        </Trans>,
       ],
     },
     {
       q: t`What is overflow?`,
       a: [
-        t`If you know how much your project needs to earn over some period of time to be sustainable, you can set a funding target with that amount. If your project earns more than that, the surplus funds are locked in an overflow pool. Anyone holding your project's tokens can claim a portion of the overflow pool in exchange for burning their tokens.`,
+        t`If you know how much your project needs to earn over some period of time to be sustainable, you can set a funding target with that amount. 
+        If your project earns more than that, the surplus funds are locked in an overflow pool. 
+        Anyone holding your project's tokens can claim a portion of the overflow pool in exchange for redeeming their tokens.`,
+        <Trans>
+          For more info, check out this {overflowVideoLink('short video')}.
+        </Trans>,
       ],
     },
     {
       q: t`What are community tokens?`,
       a: [
-        t`Each project has its own tokens. Everyone who funds a project gets a newly minted supply of tokens in return. Token balances will be tracked by the protocol until ERC-20 tokens are issued by the project owner`,
+        <Trans>
+          Each project has its own{' '}
+          <a
+            href="https://www.youtube.com/watch?v=cqZhNzZoMh8&ab_channel=SimplyExplained"
```

Comment:
**suggestion**: WDYT about linking to the official docs?

https://ethereum.org/en/developers/docs/standards/tokens/erc-20/

Reply:
Personally, I think that people who don't already know what an ERC-20 token is would rather watch a video like I've attached that read through technical docs. Wdyt?

Reply:
sgtm